pkgs/poetry/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "poetry"
-version = "1.1.11"
+version = "1.2.0a2"
```

Comment:
Why upgrade to a [testing release](https://python-poetry.org/blog/announcing-poetry-1.2.0a2/)?

Reply:
It's not meant to be merged now, just so that we can start playing with it